packages/api/migrations/20211015015859-create-domain.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
I think this file is being created with this PR: https://github.com/the-difference-engine/ksf/pull/377/files#diff-d4b5e00d803fea43842598789e4974941d62862a80fdb4f7409a39bd389f23ff so we probably shouldn't add it here as well

Reply:
File was already merged with develop, so I added the migration back.  After adding the migration the test passed.  Please advise